default-python-3.rst

```diff
@@ -0,0 +1,14 @@
+.. post:: Jan 11th, 2019
+   :author: Anthony
+
```

Comment:
```suggestion
   :tags: python

```

By adding the python tag, this ensures that the post gets put into Planet Python's RSS feed.

Reply:
I like the idea of "going global" with this post.